### PR TITLE
got the matrix numbers

### PR DESCRIPTION
**Pull request by dllauro**

### CODE REVIEW

File: answer.rb [hunk 1]
```diff
@@ -1,6 +1,23 @@
...
+y_ld = 0
+total_ld = 0
 
-matrix = first_number.times.map{ |x| gets.split(" ").map(&:to_i) }
+x_rd = 0
+y_rd = -1
+total_rd = 0
 
+matrix = first_number.times.map{ |x| gets.split(" ").map(&:to_i) }
+matrix.length.times do
```
Comment (abunsen) [inline answer.rb]: Maybe use `first_number`

File: answer.rb [hunk 2]
```diff
@@ -1,6 +1,23 @@
...
+  matrix[x_ld][y_ld] += total_ld
+  x_ld += 1
+  y_ld +=1
+end
+matrix.length.times do
```
Comment (abunsen) [inline answer.rb]: Same as above.

File: answer.rb [hunk 3]
```diff
@@ -1,6 +1,23 @@
...
+  matrix[x_rd][y_rd] += total_rd
+  x_rd += 1
+  y_rd +=1
```
Comment (RodneyPerez) [inline answer.rb]: your y_ld should go in the other direction

Comment (readytocode123) [inline answer.rb]: Look beyond what you see.

Comment (readytocode123) [inline answer.rb]: Your counter needs to get lower since this is the array that is going right to left.

File: answer.rb [hunk 4]
```diff
@@ -1,6 +1,23 @@
...
+end
+p total
```
Comment (jmopr) [inline answer.rb]: total = total_rd + total_ld

File: answer.rb (hunk 3 above)
Comment (readytocode123) [inline answer.rb]: The column counter needs to be set to n-1 first and then decrease by 1.

File: answer.rb [hunk 5]
```diff
@@ -1,6 +1,23 @@
 first_number = gets.chomp.to_i
 
+x_ld = 0
+y_ld = 0
+total_ld = 0
 
-matrix = first_number.times.map{ |x| gets.split(" ").map(&:to_i) }
+x_rd = 0
+y_rd = -1
```
Comment (readytocode123) [inline answer.rb]: The column counter needs to be set to n-1 first and then decrease by 1.

**Pull request merged**